_pytest/config.py

```diff
@@ -399,13 +399,15 @@ def consider_conftest(self, conftestmodule):
             self.consider_module(conftestmodule)
 
     def consider_env(self):
-        self._import_plugin_specs(os.environ.get("PYTEST_PLUGINS"))
+        specs = os.environ.get("PYTEST_PLUGINS")
+        if specs:
+            plugins = specs.split(',')
```

Comment:
Also, I think that all those places with handle the different ways to specify a plugin are getting unhandy to maintain.

```python
if specs:
       specs = specs.split(',')
```

And:

```python
if isinstance(plugins, str):
    plugins = [plugins]
```

Should we add a 

```python
def _get_plugins_as_list(specs: Union[str, None, List[str])) -> List[str]
``` 

Function and use it everywhere? Some subtle bugs have crept in because this logic is implemented everywhere, sometimes incompletely.